test/src/stacktrace.spec.ts

```diff
@@ -18,12 +18,19 @@ import assert from 'assert';
 
 import expect from 'expect';
 
-import {getTestState} from './mocha-utils.js';
+import {getTestState, setupTestBrowserHooks} from './mocha-utils.js';
 import {waitEvent} from './utils.js';
 
 const FILENAME = __filename.replace(/[/\-\\^$*+?.()|[\]{}]/g, '\\$&');
+const parseStackTrace = (stack: string): string => {
```

Comment:
Makes this test less reliant on the file as adding a new line somewhere will cause it to brake;
@jrandolf PTAL